documents/description.md

```diff
@@ -1,9 +1,41 @@
+
 
 ## Language design
+*If you had to capture your DSL's design in one sentence, what would it be? What constitutes a program in your language? What happens when a program runs? What kinds of input might a program take, and what kinds of output might it produce? Are there data or control structures that you know will be useful? What kinds of things might go wrong in a program in this domain (e.g., syntax errors, compile-time errors, run-time errors)? How might you design your language to prevent such errors or to clearly communicate the results of errors to the user?
+
+Note: You can describe the computations in the description.md file, or put them in separate files. Just be clear about where they are and what they mean.*
+
+My DSL's design would be: "Building yourself a wine cellar and rummaging through it to learn from your own personal tasting notes." My project has a lot of different components, but the main DSL in the project is the format of the wine tasting notes. A program would be an entry into the cellar, in the format of the wine name and the tasting notes. I haven't decided on the exact format of the notes, but they would involve things like name, varietal, location tasted, personal wine rating, primary/secondary/tertiary aromas and flavors, and additional notes. When a program like this runs, it would be put into the database along with a link to the particular type of wine from a different database. This other database would contain professional information about the wine.
+
+Another program in the language would be a user querying the database and comparing their notes. This would involve the user selecting their wine, a query to the other database to retrieve professional information about the wine, then then conversion of that information into the same format as the user's tasting notes. 
+
+A sample program in the language would take arguments like I described above (personal rating, primary aromas, etc) and the output would be to store the wine in the user's wine cellar/database. Another potential program output from this language would be in the event of a wine comparison. The user would be able to compare a wine in their cellar with professional notes, so the output of the comparison would be the professional notes formatted to look like the user's notes. 
```

Comment:
What is the information the user will glean from this? And how will it be useful to them in this domain? Basically, what is the question or problem that this functionality is solving?

If it is solving the question of only, what did others think of this wine, then this seems to mostly be a web scraper that nicely formats data. 
If it is trying to solve which proffessional their taste most allign to, I don't think this goes far enough. 

So I think you should consider whether this is a note taking DSL or a DSL to help users develop and explore different wines tastes. (Because if you already have all the data from the user and all the data from the professionals, I feel like your DSL should try and interpret the data for the user, making it more of a "computational" language.) Basically, I'm not convinced the description you provide really describes a DSL instead of a data formatter?